packages/loopring_v3/contracts/impl/BlockVerifier.sol

```diff
@@ -109,8 +108,7 @@ contract BlockVerifier is ReentrancyGuard, IBlockVerifier
         view
         returns (bool)
     {
-        bool dataAvailability = needsDataAvailability(blockType, onchainDataAvailability);
-        Circuit storage circuit = circuits[dataAvailability][blockType][blockSize][blockVersion];
+        Circuit storage circuit = circuits[onchainDataAvailability][blockType][blockSize][blockVersion];
```

Comment:
should we remove `blockType` or we keep it so we still have the flexibility of adding more types?

Reply:
Would keep it for now, doesn't hurt to have it just in case we indeed want to do that.